.reuse/dep5

```diff
@@ -155,3 +155,7 @@ License: CC-BY-ND-4.0
 Files: editors/vscode/pnpm-lock.yaml
 Copyright: Copyright © SixtyFPS GmbH <info@slint.dev>
 License: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+Files: editors/vscode/biome.json
```

Comment:
You could move that as an extra file in the definition above. They are closely related.